front/src/common/Collapsable.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useCallback, useEffect, useState, type FC, type HTMLAttributes } from 'react';
+
+import { ChevronDown, ChevronUp } from '@osrd-project/ui-icons';
+import cx from 'classnames';
+import { omit } from 'lodash';
+import { useTranslation } from 'react-i18next';
+
+interface CollapsableProps extends Omit<HTMLAttributes<unknown>, 'children' | 'collapsed'> {
+  children: [JSX.Element, JSX.Element];
+  collapsed: boolean;
+}
+const Collapsable: FC<CollapsableProps> = (props) => {
+  const { children, collapsed, className } = props;
+  const { t } = useTranslation('common');
+  const [isCollapsed, setIsCollapsed] = useState(collapsed);
+
+  const toggle = useCallback(() => {
+    setIsCollapsed((prev) => !prev);
+  }, []);
+
+  useEffect(() => {
+    setIsCollapsed(collapsed);
+  }, [collapsed]);
+
+  return (
+    <div
+      className={cx(className, 'p-1')}
+      {...omit(props, ['children', 'defaultCollapsed', 'className'])}
+    >
+      <div className="collapse-header d-flex align-items-start">
+        <div className="flex-grow-1">{children[0]}</div>
+        <button
+          type="button"
+          className="flex-grow-0 px-1 pb-1 mx-1"
+          title={
+            isCollapsed ? t('common/common:actions.expand') : t('common/common:actions.collapse')
```

Comment:
Seems like the trad doesn't work with this syntax.

```suggestion
            isCollapsed ? t('actions.expand') : t('actions.collapse')
```

And `const { t } = useTranslation('common/common');` at the top